resources/openapi_v2.yaml

```
@@ -7,20 +7,23 @@ info:
 paths:
   /applications:
     get:
-      description: Return all the applications related to a user or the apointted ministry
+      description: Return all the applications related to a user or the appointed ministry using the authentication token and admin or ministry can query the application by user-id or ministry-id and user can query the application using draft.
       parameters:
         - in: query
           name: user-id
+          description: Used to query application related to a specific user(privileges for access are set to ministries and adminstrators only).
```

```suggestion
          description: Used to query application related to a specific user (privileges for access are set to ministries and adminstrators only)
```
I see some flaws in the descriptions of the parameters. More than talk about what they can be used for it should be explaining what they are. This issue is persistent throughout the API